features/run/shared-services.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Shared service directories
+
+  As an Exosphere developer
+  I want to be able to reuse certain files in every service
+  So that I don't have to duplicate code
+
+  Rules:
+  - Shared directories are listed under `shared-directories` in `application.yml`
+  - Each shared directory is copied into every other service directory
+
+
+  Scenario: shared directories are copied into service directories
+    Given I am in the root directory of the "shared-directory" example application
+    And starting "exo run" in my application directory
+    Then it prints "shareddirectory_service_1 exited with code 0" in the terminal
```

Comment:
How about we make the shared directory print out the contents of the shared file and that is what we check for here? We can update the shared file to contain the text "shared content"